backend/src/controllers/organizationController.js

```
@@ -59,17 +68,23 @@ const createTeamWithMembers = async (req, res) => {
   }
 };
 
+
+
 const getOrganizationTeams = async (req, res) => {
   try {
     const organizationId = req.user.organizationId;
-    const result = await organizationService.getOrganizationTeams(organizationId);
+    const { page = 1, limit = 10 } = req.query; 
+    const offset = (parseInt(page) - 1) * parseInt(limit);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)